linux_os/guide/services/ssh/ssh_server/sshd_use_approved_macs/rule.yml

```diff
@@ -42,13 +46,15 @@ references:
     disa@rhel6: 2449,2450,68,803
     cis: 5.2.12
     cui: 3.1.13,3.13.11,3.13.8
-    disa: 68,803,1453
+    disa: 68,803,1453,877,3123
     hipaa: 164.308(b)(1),164.308(b)(2),164.312(e)(1),164.312(e)(2)(i),164.312(e)(2)(ii),164.314(b)(2)(i)
-    nist: AC-17(b),AC-17(2),IA-7,SC-13
+    nist: AC-17(b),AC-17(2),IA-7,SC-13,MA-4(c),MA-4(6)
```

Comment:
[MA-4(c)](https://nvd.nist.gov/800-53/Rev4/control/ma-4) is about strong authenticators vs crypto. Can you elaborate on the reasoning for adding that mapping?

Reply:
We copied these over from our STIG. Apparently we have some questionable references there that will need double-checking.